<launch>
  <!-- camera driver -->
  <node name="dvs_samsung_ros" pkg="dvs_samsung_ros" type="publisher" output="screen">
  </node> 

  <!-- visualization -->
  <node name="dvs_renderer" pkg="dvs_renderer" type="dvs_renderer">
    <param name="display_method" value="grayscale"/>
    <remap from="events" to="/dvs/events" />
    <remap from="camera_info" to="/dvs/camera_info" />
  </node>

  <!-- calib node -->
  <node name="dvs_calibration" pkg="dvs_calibration" type="mono" output="screen">
    <remap from="events" to="/dvs/events" />
    <remap from="camera_info" to="/dvs/camera_info" />
    <remap from="set_camera_info" to="/dvs/set_camera_info" />

    <param name="blinking_time_us" value="1000" />
    <param name="blinking_time_tolerance_us" value="5000" />
    <param name="enough_transitions_threshold" value="40" />
    <param name="minimum_transitions_threshold" value="10" />

    <param name="dots_w" value="5"/>
    <param name="dots_h" value="5"/>
    <param name="dot_distance" value="0.05"/>
  </node>

  <node name="rqt_gui" pkg="rqt_gui" type="rqt_gui" args="--perspective-file $(find dvs_calibration_gui)/rqt/IntrinsicCalibration.perspective" />

</launch>
